vault/resource_pki_secret_backend_root_cert.go

```diff
@@ -53,9 +57,38 @@ func pkiSecretBackendRootCertResource() *schema.Resource {
 				return e
 			}
 
-			cert, err := getCACertificate(client, d.Get(consts.FieldBackend).(string))
-			if err != nil {
-				return err
+			var cert *x509.Certificate
+			isIssuerAPISupported := provider.IsAPISupported(meta, provider.VaultVersion111)
+			if isIssuerAPISupported {
+				// get the specific certificate for issuer with issuer_id
+				cert, e = getIssuerPEM(client, d.Get(consts.FieldBackend).(string), d.Get(consts.FieldIssuerID).(string))
+				if e != nil {
+					// Check if this is an out-of-band change on the issuer
+					if util.Is500(e) && util.ErrorContainsString(e, issuerNotFoundErr) {
+						log.Printf("issuer deleted out-of-band. re-creating root cert")
```

Comment:
Should there be a level associated with this log?

Reply:
Added a `WARN` level, thanks for the catch!